rfc/wip-tform/index.md

```diff
@@ -0,0 +1,679 @@
+
+<dl>
+  <dt><strong>path</strong></dt>
+  <dd>  The path to an array containing the affine parameters.
+	The array at this path MUST be 2D whose shape MUST be `N x N`.</dd>
+  <dt><strong>rotation</strong></dt>
+  <dd> 	The  parameters stored in JSON. The matrix MUST be stored as a 2D nested array where the outer array MUST be length `N`
+  and the inner arrays MUST be length `N`.</dd>
+</dl>
+
+
+#### <a name="inverseOf">inverseOf</a>
+
+An `inverseOf` transformation contains another transformation (often non-linear), and indicates that
+transforming points from output to input coordinate systems is possible using the contained transformation.
+Transforming points from the input to the output coordinate systems requires the inverse of the contained
+transformation (if it exists).
+
+<div class=note>
+    Software libraries that perform image registration often return the transformation from fixed image
+    coordinates to moving image coordinates, because this "inverse" transformation is most often required
+    when rendering the transformed moving image. Results such as this may be enclosed in an `inverseOf`
+    transformation. This enables the "outer" coordinate transformation to specify the moving image coordinates
+    as `input` and fixed image coordinates as `output`, a choice that many users and developers find intuitive.
+</div>
+
+
+
+#### <a name="sequence">sequence</a>
+
+A `sequence` transformation consists of an ordered array of coordinate transformations, and is invertible if and only if every
+coordinate transform in the array is invertible. To apply a sequence transformation to a point in the input coordinate system,
+apply the first transformation in the list of transformations. Next, apply the second transformation to the result. Repeat until
+every transformation has been applied. The output of the last transformation is the result of the sequence.
+
+The transformations included in the `transformations` array may omit their `input` and `output` fields under the conditions
+outlined below:
+
+- The `input` and `output` fields MAY be omitted for the following transformation types: 
+    - `identity`, `scale`, `translation`, `rotation`, `affine`, `displacements`, `coordinates`
+- The `input` and `output` fields MAY be omitted for `inverseOf` transformations if those fields may be omitted for the
+    transformation it wraps
+- The `input` and `output` fields MAY be omitted for `bijection` transformations if the fields may be omitted for 
+    both its `forward` and `inverse` transformations
+- The `input` and `output` fields MAY be omitted for `sequence` transformations if the fields may be omitted for
+    all transformations in the sequence after flattening the nested sequence lists.
+- The `input` and `output` fields MUST be included for transformations of type: `mapAxis`, and `byDimension` (see the note
+    below), and under all other conditions.
+
+
```

Comment:
Not clear to me. Also I'd use list instead of array since array is used to represent the component of a single transformation.